bundles/org.eclipse.swt/META-INF/MANIFEST.MF

```diff
@@ -2,7 +2,7 @@ Manifest-Version: 1.0
 Bundle-Name: %pluginName
 Bundle-Vendor: %providerName
 Bundle-SymbolicName: org.eclipse.swt; singleton:=true
-Bundle-Version: 3.124.100.qualifier
+Bundle-Version: 3.125.100.qualifier
```

Comment:
Recall I said increment the third segment but here you incremented the second segment.

You'll probably want  to read this:

https://wiki.eclipse.org/Version_Numbering